ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/lvm.py

```diff
@@ -369,14 +369,12 @@ def makeLVMap(self, columns):
             lv_om.modname = 'ZenPacks.zenoss.LinuxMonitor.SnapshotVolume'
         elif columns['lv_metadata_size']:
             lv_om.id = 'tp-{}'.format(self.prepId(columns['vg_name'])+'_'+self.prepId(columns['lv_name']))
-            lv_om.metadatasize = columns['lv_metadata_size']
+            lv_om.metadatasize = int(columns['lv_metadata_size'])
             lv_om.relname = 'thinPools'
             lv_om.modname = 'ZenPacks.zenoss.LinuxMonitor.ThinPool'
         else:
             if columns['pool_lv']:
                 lv_om.set_thinPool = 'tp-{}'.format(self.prepId(columns['vg_name'])+'_'+self.prepId(columns['pool_lv']))
-            else:
-                lv_om.set_thinPool = ''
```

Comment:
What problem did this cause? Maybe a better approach is `lm_om.set_thinPool = None` instead of setting it to an empty string in this case?

Reply:
I thought that this was causing a remodel, but it seemed unneccesary otherwise.